modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/IgniteH2Indexing.java

```diff
@@ -592,7 +594,14 @@ private GridQueryFieldsResult executeSelectLocal(
 
```

Comment:
On the line above you create a list of argument from an array of parameter. The only usage of this list is on the 599 line where you create an array from it. So why don't just use the original array from `qryParams.arguments()`?

Reply:
Good point, thx.